dags/prio.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator, GKEClusterCreateOperator, GKEClusterDeleteOperator
+from utils.gke import create_gke_config
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.contrib.operators.s3_to_gcs_transfer_operator import S3ToGoogleCloudStorageTransferOperator # noqa
+
+DEFAULT_ARGS = {
+    'owner': 'hwoo@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 5, 31),
+    'email': ['hwoo@mozilla.com', 'dataops+alerts@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=15),
+}
+
+cluster_name = 'gke-prio-a'
+
+gcp_conn_id = "google_cloud_prio_a"
+connection = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id)
+
+cluster_def = create_gke_config(
+    name=cluster_name,
+    service_account="prio-runner-a@moz-fx-priotest-project-a.iam.gserviceaccount.com",
+    owner_label="hwoo",
+    team_label="dataops"
+    )
+
+gke_dag = DAG(
+    'gke_prio_a',
+    default_args=DEFAULT_ARGS,
+    dagrun_timeout=timedelta(hours=6),
+    schedule_interval='0 2 * * *'
+)
+
+create_gke_cluster = GKEClusterCreateOperator(
+    task_id='create_gke_cluster',
+    project_id=connection.project_id,
+    location='us-west1-b',
+    gcp_conn_id=gcp_conn_id,
+    body=cluster_def,
+    dag=gke_dag)
+
+
+run_prio = GKEPodOperator(
+    task_id='run_prio_a',
+    gcp_conn_id=gcp_conn_id,
+    project_id=connection.project_id,
+    location='us-west1-b',
+    cluster_name=cluster_name,
+    name='run-hello-world',
+    namespace='default',
+    image='hello-world:latest',
```

Comment:
You can use "mozilla/prio:latest" and run "scripts/test-cli-integration" as the command if you want a more reasonable set of logs to test with. 

https://github.com/mozilla/python-libprio/blob/e8d4fc07bf3ae0059af6218673e655285a62bb09/.circleci/config.yml#L66